colin/core/loader.py

```diff
@@ -0,0 +1,17 @@
+import inspect
+
+from importlib.util import module_from_spec
```

Comment:
Unfortunately, the [approach from atomic](https://github.com/projectatomic/atomic-reactor/blob/master/atomic_reactor/plugin.py#L123) is deprecated... (The `imp` module.)

This will probably work only for python3.5+...

Reply:
damn

Reply:
Can I leave this for now and look into that later?

Reply:
sounds good to me